lib/xcov/options.rb

```diff
@@ -140,6 +140,16 @@ def self.available_options
           is_string: false,
           default_value: false
         ),
+        FastlaneCore::ConfigItem.new(
+          key: :slack_username,
+          description: "The username which is used to publish to slack",
+          optional: true
+        ),
+        FastlaneCore::ConfigItem.new(
+          key: :slack_message,
+          description: "The message which is published together with a successful report",
+          optional: true
+        ),
```

Comment:
I would add a `default_value` argument with `xcov` as value for `slack_username` and `Your *xcov* coverage report` for `slack_message` options. That would simplify the logic within `slack_poster.rb` :)